File: dmpnn-tox24-leaderboard/train-the-test/rmse.R
library(readr)

args <- commandArgs(trailingOnly = TRUE)
input_file <- args[1]

data <- read_csv(input_file)

actual_values <- data$activity_scaled
predicted_values <- data$predicted

rmse <- sqrt(mean((actual_values - predicted_values)^2))

# Step 4: Print the RMSE value
print(paste("The RMSE is:", rmse))

File: dmpnn_blindtest_tox24_prediction/prediction/rmse.py
import pandas as pd
import numpy as np
from sklearn.metrics import mean_squared_error
import sys

# Load the merged dataset (after combining actual and predicted values)
merged_df = pd.read_csv(sys.argv[1])

# Extract the 'actual' and 'predicted' columns
actual_values = merged_df['actual']
predicted_values = merged_df['predicted']

# Calculate RMSE
rmse = np.sqrt(mean_squared_error(actual_values, predicted_values))

# Print the RMSE
print(f"Root Mean Squared Error (RMSE): {rmse}")

